lib/cisco_node_utils/cmd_ref/encapsulation.yaml

```diff
@@ -0,0 +1,22 @@
+# encapsulation profile
+# encapsulation profile feature is available only on n7k
+---
+_exclude: [N3k, N5k, N6k, N9k]
+
+all_encaps:
+  multiple: true
+  get_command: "show encapsulation profile"
+  get_value: '/^encapsulation profile vni (\S+)/'
+
+create:
+  set_value: "encapsulation profile vni %s"
+
+destroy:
+  set_value: "no encapsulation profile vni %s"
+
+dot1q_map:
```

Comment:
Will users understand what `dot1q_map` means in this case? versus `vni` or `dot1q_vni`. I'm not saying you need to change this, just asking if this is a keyword that users can relate to; in other words when you write the readme doc entry for this property will the description say 'this is the dot1q mapping between a vlan and vni' or 'this is the profile vni' or...? You should also check the yang model to see what they use for this attribute.